### PR TITLE
feat(ui5-calendar): update visual design to latest SAP Horizon spec

### PR DESCRIPTION
**Pull request by Todor-ads**
Related to: https://github.com/SAP/ui5-webcomponents/issues/6999

**Before** 

**After**

### CODE REVIEW

**Comment by ilhan007** [comment]
Could you describe the update in the Pull Request description, adding screenshot would be also nice

File: packages/main/src/themes/sap_horizon_hcb/DayPicker-parameters.css [hunk 1]
```diff
@@ -28,7 +28,11 @@
...
 	--_ui5_daypicker_item_now_selected_between_border: 0.0625rem solid var(--sapContent_Selected_ForegroundColor);
 	--_ui5_daypicker_item_now_selected_between_border_radius: 0.1875rem;
-	--_ui5_daypicker_item_selected_text_outline: 0.0625rem solid var(--sapContent_FocusColor);
+	--_ui5_daypicker_item_selected_text_outline: 0.125rem solid var(--sapSelectedColor);
 	--_ui5_daypicker_item_select_between_border: 1px solid var(--sapContent_Selected_ForegroundColor);
-	--_ui5_daypicker_item_now_inset: 0.375rem;
+	--_ui5_daypicker_item_now_inset: 0.1875rem;
+	--_ui5_daypicker_item_selected__secondary_type_text_outline: 0.125rem solid var(--sapSelectedColor);
+	--_ui5_daypicker_item_now_box_shadow: inset 0 0 0 0.0625rem var(--_ui5_daypicker_item_selected_border_color);
+    --_ui5_daypicker_two_calendar_item_selected_now_border_radios_foucs: 0.3125rem;
```
Comment (ilhan007) [inline packages/main/src/themes/sap_horizon_hcb/DayPicker-parameters.css:36]: foucs -> focus

File: packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css [hunk 2]
```diff
@@ -34,5 +34,13 @@
 	--_ui5_daypicker_item_now_selected_between_border_radius: 0.1875rem;
 	--_ui5_daypicker_item_select_between_border: 1px solid var(--sapContent_Selected_ForegroundColor);
 	--_ui5_daypicker_item_now_inset: 0.375rem;
-	--_ui5-dp-item_withsecondtype_border: 0.1875rem;
+	--_ui5-dp-item_withsecondtype_border: 0.25rem;
+	--_ui5_daypicker_item_selected__secondary_type_text_outline: 0.125rem solid var(--sapSelectedColor);
+	--_ui5_daypicker_two_calendar_item_now_day_text_contetn: "";
```
Comment (ilhan007) [inline packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css:39]: contetn -> content

File: packages/main/src/themes/sap_horizon/DayPicker-parameters.css [hunk 3]
```diff
@@ -36,6 +36,14 @@
 	--_ui5_daypicker_item_weeekend_filter: brightness(105%);
 	--_ui5_daypicker_item_selected_hover: var(--sapList_Hover_Background);
 	--_ui5_daypicker_item_now_inset: 0.375rem;
-	--_ui5-dp-item_withsecondtype_border: 0.1875rem;
+	--_ui5-dp-item_withsecondtype_border: 0.25rem;
+	--_ui5_daypicker_item_selected__secondary_type_text_outline: 0.125rem solid var(--sapSelectedColor);
+	--_ui5_daypicker_two_calendar_item_now_day_text_contetn: "";
+	--_ui5_daypicker_two_calendar_item_now_selected_border_width:  0.1875rem;
+	--_ui5_daypicker_two_calendar_item__border_redius: 0.5rem;
```
Comment (ilhan007) [inline packages/main/src/themes/sap_horizon/DayPicker-parameters.css:43]: redius -> radius

Comment (ilhan007) [inline packages/main/src/themes/sap_horizon/DayPicker-parameters.css:43]: item__border -> item_border - just one underscore is enough

File: packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css [hunk 4]
```diff
@@ -34,5 +34,13 @@
 	--_ui5_daypicker_item_now_selected_between_border_radius: 0.1875rem;
 	--_ui5_daypicker_item_select_between_border: 1px solid var(--sapContent_Selected_ForegroundColor);
 	--_ui5_daypicker_item_now_inset: 0.375rem;
-	--_ui5-dp-item_withsecondtype_border: 0.1875rem;
+	--_ui5-dp-item_withsecondtype_border: 0.25rem;
```
Comment (ilhan007) [inline packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css:37]: In some cases "dp" is used, in other "daypicker", I suggest use the shorter everywhere

File: packages/main/src/themes/base/DayPicker-parameters.css [hunk 5]
```diff
@@ -45,4 +45,13 @@
 	--_ui5_daypicker_item_selected_hover: var(--sapContent_Selected_Hover_Background);
 	--_ui5_daypicker_item_border_radius_item: none;
 	--_ui5_daypicker_item_now_selected_outline_offset: -0.3125rem;
+	--_ui5_daypicker_two_calendar_item_now_inset: 0.1875rem;
+	--_ui5_daypicker_item_selected__secondary_type_text_outline: none;
+	--_ui5_daypicker_two_calendar_item_now_day_text_contetn: none;
+	--_ui5_daypicker_two_calendar_item_now_selected_border_width: 0.0625rem;
+	--_ui5_daypicker_two_calendar_item__border_redius: 0.0625rem;
+	--_ui5_daypicker_two_calendar_item_no_selected_inset: 0.125rem;
+    --_ui5_daypicker_two_calendar_item_selected_now_border_radios_foucs: 0.0625rem;
```
Comment (ilhan007) [inline packages/main/src/themes/base/DayPicker-parameters.css:54]: radios -> radius 
foucs -> focus

Please check all the text with some grammar/syntax tool

**Review by ilhan007 — CHANGES_REQUESTED** [review]
You can find my comments inline. In addition could you add some snapshots before/after so everyone can see the update visually.

File: packages/main/src/themes/DayPicker.css [hunk 6]
```diff
@@ -277,3 +277,40 @@
...
+	border-width: 0.125rem;
+}
+
+:host([secondary-calendar-type]) .ui5-dp-item:focus::after {
+	inset: var(--_ui5_daypicker_two_calendar_item_no_selected_inset);
+	border-radius: var(--_ui5_daypicker_two_calendar_item__border_redius);
+}
+
+:host([secondary-calendar-type]) .ui5-dp-item.ui5-dp-item--now::after {
+    /* //--_ui5_daypicker_two_calendar_item_no_selected_inset */
```
Comment (ilhan007) [inline packages/main/src/themes/DayPicker.css:293]: commented code

File: packages/main/src/themes/sap_horizon_hcb/DayPicker-parameters.css (hunk 1 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon_hcb/DayPicker-parameters.css:36]: done

File: packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css (hunk 2 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css:39]: done

File: packages/main/src/themes/sap_horizon/DayPicker-parameters.css (hunk 3 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/DayPicker-parameters.css:43]: done

Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/DayPicker-parameters.css:43]: done

File: packages/main/src/themes/DayPicker.css (hunk 6 above)
Comment (Todor-ads) [reply packages/main/src/themes/DayPicker.css:293]: done

File: packages/main/src/themes/base/DayPicker-parameters.css (hunk 5 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/DayPicker-parameters.css:54]: done

File: packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css (hunk 4 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css:37]: I agree with you. I propose that this happen in a separate change that I am preparing and I will give a reference to it here

**Comment by ilhan007** [comment]
Unrelated test is failing, I reported it to @unazko, please discuss it within the team and hopefully find a way to make it stable:

Value change is applied.: expected 'Sha. 24, 1443 AH, 10:27:26 AM' to equal 'Sha. 17, 1443 AH, 10:27:26 AM'

File: packages/main/src/themes/DayPicker.css [hunk 7]
```diff
@@ -207,16 +207,16 @@
 
 .ui5-dp-item.ui5-dp-item--selected.ui5-dp-item--now:focus::after {
 	inset: var(--_ui5_daypicker_item_now_inset);
-	border-radius: var(--_ui5_day_picker_item_selected_now_border_radios_foucs);
-	border: var(--_ui5_day_picker_item_selected_now_border_foucs);
+	border-radius: var(--_ui5_day_picker_item_selected_now_border_radios_focus);
```
Comment (ilhan007) [inline packages/main/src/themes/DayPicker.css:210]: radi**o**s_focus -> radi**u**s_focus

**Comment by ilhan007** [comment]
Besides the screenshots, you can give all some context that for example this change is to update the visual design **to the latest Horizon spec**, so that all reviewers understand the motivation behind the change

**Comment by ilhan007** [comment]
Besides the screenshots, you can give all some context that for example this change is to update the visual design **to the latest Horizon spec**, so that all reviewers understand the motivation behind the change

**Comment by ilhan007** [comment]
@Todor-ads From code perspective the issues are resolved, thank you.
But additional review from the team is required. Also, I did not check if it works functionally in all themes, only commented the code issues

**Comment by petyabegovska** [comment]
Related to: #6999

**Review by NHristov-sap — APPROVED** [review]

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**